source/components/organisms/SummaryList/SummaryList.tsx

```diff
@@ -273,6 +280,7 @@ const SummaryList: React.FC<Props> = ({
 
   itemIdmap.forEach((value) => {
     if (Array.isArray(value.answers)) {
+      let lastCategory: string | undefined = "";
```

Comment:
Are you sure that you need the `undefined` as a type here? You are defaulting it to an empty string.